chapters/statemachines.tex

```diff
@@ -30,10 +30,9 @@ \chapter{State Machines}\label{state-machines}
   in Statecharts or in Lucid Synchrone state machines but only
   indirectly with appropriate conditions on transitions.
 \item
-  Modelica blocks can be used as states. They might contain
-  clocked or clocked discretized continuous-time equations (in the
-  latter case, the equations are integrated between the previous and the
-  next clock tick, if the corresponding state is active).
+  Modelica blocks can be used as states.
+  They might contain clocked equations.
+  If the equations are discretized they are integrated between the previous and the next clock tick, if the corresponding state is active.
```

Comment:
Not really related to this PR, but what about the current clock tick?  Is it skipped?

Reply:
That is a bad text, will correct it separately. Clearly the current clock tick should be included. The most forgiving interpretation is that "next" means "next (after previous)" - which is the same as "current".

Reply:
#3458